modules/ROOT/pages/as2/as2-connectorMule4-60.adoc

```diff
@@ -0,0 +1,80 @@
+= AS2 Connector - Get Started - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium]
+
+AS2 Connector Version 6.0.0
+
+Mule runtime engine (Mule) supports the AS2 protocol through the Anypoint Connector for AS2 (AS2 Connector).
+
+The AS2 Connector enables trading partners to transmit and receive AS2 messages with EDI-X12, EDIFACT, XML, or binary payloads. For example:
+
+* Send messages, signed or unsigned, over HTTP or HTTPS, following the AS2 protocol
+
+* Receive messages, also over HTTP or HTTPS, following the AS2 protocol
+
+* Receive asynchronous MDN for the message sent
+
+* Storing signed copies of receipts and original message exchanges for non-repudiation purposes
+
+Release Notes: xref:release-notes::connector/as2-connector-release-notes-mule-4.adoc[AS2 Connector Release Notes]
+
+== About Connectors
+
+Anypoint connectors are Mule runtime engine extensions that enable you to connect
+to APIs and resources on external systems, such as X12 and RosettaNet.
+
+== Prerequisites
+
+Before creating an app, you must have access to the AS2 target resource and
+Anypoint Platform. You also need understand how to create a Mule app using Anypoint Studio.
+You should have familiarity with AS2.
+
+=== Considerations when Migrating from the AS2 connector 5.x
```

Comment:
This topic is subordinate to Prerequisites?
I'd say that this .adoc needs reorganization.

Reply:
```suggestion
=== Considerations When Migrating from AS2 Connector 5.x
```